### ISSUE TITLE
Did Can I Use?

### ISSUE THREAD

**Issue by tyurik1** [RavingRavenpaw/ctOS#1]
so I search Toooo very Search User Of Develop Batch file but Only Show my f**king central and this ctOS. so can I Fork it?

**Comment by RavingRavenpaw** [comment]
@tyurik1 Yeah, you can do whatever you want with this! This was just a silly little project I did back in middle school, honestly.
